code/modules/events/vent_clog.dm

```diff
@@ -9,27 +9,17 @@
 		"water",
 		"carbon",
 		"flour",
-		"radium",
-		"toxin",
-		"cleaner",
-		"nutriment",
-		"condensedcapsaicin",
-		"mindbreaker",
-		"lube",
-		"plantbgone",
-		"banana",
-		"space_drugs",
-		"holywater",
-		"ethanol",
-		"hot_coco",
-		"sacid",
-		"hyperzine",
-		"paint",
-		"luminol",
-		"fuel",
 		"blood",
-		"sterilizine",
-		"ipecac"
+		"cornoil",
+		"milk",
+		"egg",
+		"butter",
+		"cream",
+		"cheesewheel",
```

Comment:
cheesewheel is not a reagent.

Reply:
Isn't it, technically, but only exists to spawn cheese wheels? That's the intention, here.

Reply:
No, it is not a reagent. It is a reaction, two different things.